### PR TITLE
Fix admin API doc formatting

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Fix admin API doc formatting

## Motivation and Context
Fix admin API doc formatting

## How Has This Been Tested?
Tested visually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: pkg/madmin/API.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ func main() {
 ```
 
 | Service operations         | Info operations  | Healing operations                    | Config operations         | Misc                                |
-|:------------------------------------|:----------------------------|:----------------------------|:--------------------------------------|:--------------------------|:------------------------------------|
-| [`ServiceStatus`](#ServiceStatus)   | [`ServerInfo`](#ServerInfo) | [`Heal`](#Heal)             | [`GetConfig`](#GetConfig) | [`SetCredentials`](#SetCredentials) |
-| [`ServiceSendAction`](#ServiceSendAction) | | | [`SetConfig`](#SetConfig) |                                     |
+|:----------------------------|:----------------------------|:--------------------------------------|:--------------------------|:------------------------------------|
```
Comment (kannappanr) [inline pkg/madmin/API.md:40]: This still doesn't look good. Maybe it needs more text?

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

File: pkg/madmin/API.md (hunk 1 above)
Comment (harshavardhana) [reply pkg/madmin/API.md:40]: It's fixed

**Review by kannappanr — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Review by ebozduman — APPROVED** [review]
LGTM

**Pull request merged**